RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractJumpLists.java

```diff
@@ -217,8 +217,14 @@ void process(Content dataSource, IngestJobContext context, DataSourceIngestModul
                                                                          TskData.EncodingType.NONE);
                                 derivedFiles.add(derivedFile);
 
-                            } catch (IOException | JLnkParserException e) {
-                                logger.log(Level.WARNING, String.format("No such document, or the Entry represented by documentName is not a DocumentEntry link file is %s", jumpListFile), e); //NON-NLS
+                            } catch (IOException | JLnkParserException ex) {
+                                logger.log(Level.WARNING, String.format("No such document, or the Entry represented by documentName is not a DocumentEntry link file is %s", jumpListFile), ex); //NON-NLS
+                            } catch (TskCoreException ex) {
+                                logger.log(Level.WARNING, String.format("Error trying to add dervived file %s", lnkFileName), ex); //NON-NLS
+                            } catch (IndexOutOfBoundsException ex) {
```

Comment:
@markmckinnon So what line of code is throwing this exception?

Reply:
The read statment from line 194 is causing the issue.